src/storage-provider-local.ts

```diff
@@ -7,6 +8,14 @@ export default class LocalStorageProvider implements IStorageProvider {
         try {
             const repo = JSON.stringify(data);
             const key = `${this.prefix}:${name}`;
+            try {
+                AsyncStorage.setItem(key, repo).then((e: any) => {
+                    console.log(`Unleash local save completed for ${key} with value ${repo} and response is ${JSON.stringify(e)}`);
+                });
+            } catch(ex) {
+                console.error(ex);  
+            }
+            // below line is not needed for react-native mobile app
             window.localStorage.setItem(key, repo);
```

Comment:
Should we just remove the `winow.localStorage` call all-together? I guess AsyncStorage is used on web as well?

Reply:
Retained this line so that the same package can be used for both web and mobile apps.

Reply:
We can remove this if you're goal is to have separate packages for web and mobile

Reply:
I believe Async-storage cannot be used for websites

Reply:
Suggestion: 
Create a new repo `unleash-proxy-client-react-native` from current `unleash-proxy-client-js` and then we can merge this PR to the newly created repo(we can remove window.localStorage as it is exclusive for react-native mobile apps)